R/mergeData.R

```diff
@@ -1,22 +1,50 @@
 #' @title Incorporating feature engineering with clinical data
 #'
+#' @description Combines the feature engineering results from \code{computeRegionEntropy()}
+#' with clinical data to form a new data set.
+#'
+#' @details The function takes in three parameters: \code{feat_eng_df} is an output
+#' of \code{computeRegionEntropy()}. \code{clinical_data} is the clinical data
+#' that one wants to merge the entropy data with. \code{id_mapping} is an optional
+#' argument that allows the practitioner to map different forms of patient
+#' identification with each other. If this is \code{null}, then it will be assumed
+#' that the 'pid' column in \code{feat_eng_df} is the same identification as what
+#' is used in \code{clinical_data}. If \code{id_mapping} is not \code{null},
+#' then 'pid' will be used as the common identification between \code{feat_eng_df}
+#' and \code{clinical_data}.
+#'
+#' @param feat_eng_df A \code{tibble} that takes the same form as an output of
+#' \code{computeRegionEntropy()}.
+#' @param test_clinical_data A \code{dataframe} that includes the clinical data
+#' of the patients.
+#' @param match A \code{character} indicating which column name corresponds to
+#' the identificaton seen in \code{feat_eng_df} and \code{id_mapping}
+#' @param id_mapping A \code{dataframe} that has the following columns:
+#' 'SubId', 'PatID', 'ScrID', and 'DataDir'.
```

Comment:
The first and last columns of this data frame are not necessary. You should think about which columns are actually used in the function and only require a data frame with those specified columns, instead of blindly going off the test data. Of course, the function should be amenable to extra columns, but it's not a requirement.